modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -1241,6 +1241,16 @@
 				# infinite loop between 0.0 and 0.99
 				f = wrapf(f + 0.1, 0.0, 1.0)
 				[/codeblock]
+				[codeblock]
+				# infinite rotation (in radians)
+				angle = wrapf(angle + 0.1, 0.0, TAU)
+				[/codeblock]
+				Note: If you want to have a simple wrap behavior from 0.0 to 'n' (where 'n' is a positive float number) then its better for performance to use [method fmod] method like [code]fmod(number, n)[/code].
+				The usage of this function is more flexible than using [method fmod] approach by giving the user a simple control over minimum value. Its also fully support negative numbers eg.
```

Comment:
Should be:

"The usage of <b>[code]wrapf[/code]</b> is more flexible than using **the** [method fmod] approach by giving the user ~~a simple~~ control over **the** minimum value. **It** also fully support**s** negative numbers<b>,</b> **e.g.**".

Reply:
Fixed